src/pages/index.js

```diff
@@ -120,6 +121,9 @@ const modalFormUser = new PopupwithForm({
       .catch(console.error)
       .finally(() => {
         modalFormUser.renderLoading(false);
+        if ("edit-modal-form" in formValidators) {
+          formValidators["edit-modal-form"].resetValidation();
+        }
```

Comment:
`resetValidation` is needed only when you open modals. Please, remove them from `finally`.